ultralytics/models/yolo/classify/val.py

```diff
@@ -29,6 +29,7 @@ class ClassificationValidator(BaseValidator):
     def __init__(self, dataloader=None, save_dir=None, pbar=None, args=None, _callbacks=None):
```

Comment:
The reason you changed classify is due to regress being based on classify?

Reply:
No, the only changes in classify val are lines of code that save incorrectly classified image names to a file. This change does not affect regress val. Regress val has analogous code to save outlying image names to a file, and these changes for regress and classify would be good to have for model accuracy analysis, I think.